packages/devtools_app/lib/src/screens/inspector/layout_explorer/box/box.dart

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// ignore_for_file: avoid_redundant_argument_values, import_of_legacy_library_into_null_safe, unnecessary_import
+// ignore_for_file: unnecessary_import
```

Comment:
let's remove this ignore and just get rid of the unnecessary import

Reply:
will resolve in other PR

Reply:
Addressed in https://github.com/flutter/devtools/pull/4011